media-libs/phonon/phonon-4.12.0-r5.ebuild

```diff
@@ -63,6 +63,6 @@ src_configure() {
 
 src_install() {
 	cmake_src_install
-	make_desktop_entry "${PN}settings" \
-		"Phonon Audio and Video" preferences-desktop-sound
+	make_desktop_entry --eapi9 "${PN}settings" -a "org.kde.${PN}settings" \
+		-n "Phonon Audio and Video" -i preferences-desktop-sound
```

Comment:
While this very likely works, i think this one might be a very good candidate for contributing a .desktop file upstream and use that in the future.